pom.xml

```diff
@@ -146,8 +146,8 @@
     <!-- from citizensnpcs repository -->
     <dependency>
       <groupId>net.citizensnpcs</groupId>
-      <artifactId>citizens</artifactId>
-      <version>2.0.27-SNAPSHOT</version>
+      <artifactId>citizens-main</artifactId>
```

Comment:
Do we actually need the `citizens-main` dependency or would `citizensapi` be sufficient? In case we do have non-API dependencies it might be worth having a try to remove them by using the appropriate API if possible (in a separate PR).